core/src/com/unciv/models/ruleset/unique/UniqueFlag.kt

```diff
@@ -5,7 +5,8 @@ import java.util.EnumSet
 enum class UniqueFlag {
     HiddenToUsers,
     NoConditionals,
-    AcceptsSpeedModifier
+    AcceptsSpeedModifier,
+    AcceptsGameProgressModifier
```

Comment:
Good enough for now, if we see we can merge these - that anything that can accept speed modifiers can also accept progress - we can always merge them later